ansible/setup.yml

```diff
@@ -40,6 +40,18 @@
         - ssh-keys
         - devops-ssh-keys
 
+    # Automatic cleanup is not supported. You must remove old keys manually.
+    - name: "Add CI master keys"
+      authorized_key:
+        user: "{{ ansible_user }}"
+        key: "{{ lookup('file', item) }}"
+      with_items:
+        - files/travis/master_key.pub
```

Comment:
Not sure where this file comes from.

Reply:
I understood - it is relative path so https://github.com/aeternity/infrastructure/blob/97bf5d80f6a739eefdc039fad5c83fe421a79ca9/ansible/files/travis/deploy_key.pub

Shall these travis-related files be deleted too?

Reply:
Nice catch, I deleted all travis keys in dd06a4e